dev_guide/routes.adoc

```diff
@@ -68,14 +57,25 @@ spec:
     kind: Service
     name: frontend
 ----
-====
-
 <1> For xref:../architecture/networking/routes.adoc#path-based-routes[path-based routing], specify a path component that can be compared against a URL.
 endif::[]
+ifdef::openshift-online[]
+Using the CLI, create an unsecured route. For {product-title} Starter, follow this example:
+
+----
+$ oc expose svc/frontend
+----
+
+For {product-title} Pro, follow this example, with `--hostname` being optional:
 
+----
+$ oc expose svc/frontend --hostname=www.example.com
+----
+endif::[]
+The new route inherits the name from the service unless you specify one
+using the `--name` option.
```

Comment:
This sentence will show up before the yaml example for Online and after for non-Online, right? It seems like a weird inconsistency, and probably it would be better to put it immediately after the comment, before the yaml, but it's not a huge deal.